client/src/components/SearchPage/SearchField.js

```diff
@@ -0,0 +1,111 @@
+import React, {Component} from 'react';
+import './SearchField.css'
+import Logo from '../assets/logo.png'
+import Search from '../assets/search.png';
+import Mic from '../assets/mic.png'
+import Suggestion from "./data";
+import data from './data'
+class SearchField extends Component{
+
+
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            words: [],
+            userInput: '',
+            matches: [],
+            rowHighlighted: -1
+        }
+    }
+
+    componentDidMount(){
+        this.setState({
+            words: data.words
+        })
+    }
+
+    handleUserInput(e) {
+        let input = e.target.value;
+        let {words} = this.state;
+        let matches = [];
+
+        if (input){
+            for (let i = 0; i < words.length; i++){
```

Comment:
Don't use this for loop Syntax. If you're gonna be looping throughout the `words` array, use `words.forEach()` instead. Improves readability and keeps the code uniform.